core/prisma/migrations/20250501133901_update_fts/migration.sql

```diff
@@ -0,0 +1,140 @@
+
+CREATE OR REPLACE FUNCTION strip_html_tags(text_with_tags text)
+RETURNS text AS $$
+BEGIN
+    RETURN regexp_replace(
+        regexp_replace(
+            regexp_replace(
+                text_with_tags,
+                '<[^>]+>', -- removes HTML tags
+                ' ',
+                'gi'
+            ),
+            '&[^;]+;', -- removes HTML entities
+            ' ',
+            'gi'
+        ),
+        '\s+', -- collapse multiple spaces
+        ' ',
+        'g'
+    );
+END;
+$$ LANGUAGE plpgsql IMMUTABLE;
```

Comment:
`immutable` means its a pure function. postgres allegedly does some stuff with that info